### DISCUSSION TITLE
docs/linux/gaming/tf2/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#8] [Giscus]
# docs/linux/gaming/tf2/

Stolen from https://store.steampowered.com/app/440/Team_Fortress_2/
Pretty OG game if I do say myself. Still can’t access the chat for some reason, I think somebody said online that I gotta buy something from the store, so I might do that.
Anyway, first you have to install lib32-gperftools
Run this line in launch options:
LD_PRELOAD=$LD_PRELOAD:/usr/lib32/libtcmalloc_minimal.so %command% -secure 

https://liluzibird.github.io/docs/linux/gaming/tf2/

**Comment by Ast3risk-ops** [comment]
I've improved on this quite a bit: https://guides.asterisk.lol/games/pc/tf2/linux/

**Reply by liluzibird** [reply]
Nice guide. Didn't know about guidebox, that looks really clean.

**Reply by Ast3risk-ops** [reply]
It's just the name of my [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) site.

**Reply by liluzibird** [reply]
No comment section huh?

**Reply by Ast3risk-ops** [reply]
Nope! Docs and guides never have them, but you can add them if you want. If someone wants to suggest something, they make a PR or issue.

**Reply by liluzibird** [reply]
I think it depends. I've seen some guides that have comment sections in case anybody has issues with something.

**Reply by Ast3risk-ops** [reply]
The Giscus integration is meant for the blog part of material for mkdocs anyway, but I might test adding one.

**Reply by Ast3risk-ops** [reply]
Added. It's not perfect but it'll do.

**Reply by Ast3risk-ops** [reply]
Nevermind, it conflicts with the privacy plugin and excluding it is an Insiders (paid) feature. Gonna remove it.
